src/common/ImageAvatar.js

```diff
@@ -15,14 +15,17 @@ const styles = StyleSheet.create({
 });
 
 export default({ avatarUrl, size, status, isCircular, onPress }) => {
-  const imageStyle = {
+  const touchableStyle = {
```

Comment:
Actually `imageStyle` you can't move as is because it references `isCircular`. Options include:
- Two different styles and then choose the style based on `isCircular` in the component
- Remove `isCircular` and make it always false